conftest.py

```diff
@@ -32,3 +35,19 @@ def pytest_addoption(parser):
     parser.addoption(
         "--noskip", action="store_true", default=False, help="Treat skipped tests as errors"
     )
+
+
+def pytest_runtestloop():
+    # These files are so big they exceed our "immutable symlink" hueristic. However, our integration
+    # tests run Pants in the sandbox, and Pants doesn't allow globbing absolute symlinks.
+    # Therefore we re-materialize them from their symlink.
```

Comment:
Is this something specific to the Pants repo, or will it apply to others too?

Reply:
It's because we run pants in a pants made sandbox, so very much a pants repo limitation